bouncer/authentication/permissions.py

```diff
@@ -13,3 +15,22 @@ def has_permission(self, request, view):
             if not request.user.has_perm("blacklist.view_emailhostentry"):
                 return False
         return True
+
+
+class PostRequestViewSetPermission(permissions.BasePermission):
```

Comment:
This can be merged with the `GetRequestViewPermission` and separate the different logic based on `request.method`